Update LurkingWifeMode.md (#186)
zkSNACKs/WasabiDoc: docs/using-wasabi/LurkingWifeMode.md

---
{
  "title": "Lurking Wife Mode",

>   "description": "A feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."

Reviewer: What about:
```suggestion
  "description": "A feature that hides sensitive and critical information from physical observers designed to allow you to post screenshots without worrying about leaking private details of your wallet. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

>   "description": "A function that hides sensitive and critical information from physical observers and allows you to post screenshots without having to worry about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."

Reviewer 2: ```suggestion
  "description": "A feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

}
---

[[toc]]

# Lurking Wife Mode (LWM)

> Lurking Wife Mode is a function of Wasabi Wallet that hides sensitive and critical information from physical observers and allows you to post screenshots without having to worry about your privacy.

Reviewer 2: ```suggestion
Lurking Wife Mode is an exclusive Wasabi Wallet feature that hides sensitive and critical information from physical observers and allows you to post screenshots without worrying about your privacy.
```

![](https://i.imgur.com/4OgtaHP.png)

> ## How to activate Lurking Wife Mode

Reviewer 2: Maybe this should be add to the FAQ?

Author: You are right! Adding it in a couple of hours
